bootstrap/center-text-with-bootstrap.html

```diff
@@ -0,0 +1,60 @@
+<link href="https://fonts.googleapis.com/css?family=Lobster" rel="stylesheet" type="text/css">
+
+    <style>
+        .red-text {
+          color: red;
+        }
+
+        h2 {
+          font-family: Lobster, Monospace;
+        }
+
+        p {
+          font-size: 16px;
+          font-family: Monospace;
+        }
+
+        .thick-green-border {
+          border-color: green;
+          border-width: 10px;
+          border-style: solid;
+          border-radius: 50%;
+        }
+
+        .smaller-image {
+          width: 100px;
+        }
+    </style>
+
+    <div class="container-fluid">
+      <h2 class="red-text text-center">CatPhotoApp</h2>
+
+      <p>Click here for <a href="#">cat photos</a>.</p>
+
+      <a href="#"><img class="smaller-image thick-green-border" src="https://cdn.freecodecamp.org/curriculum/cat-photo-app/relaxing-cat.jpg" alt="A cute orange cat lying on its back."></a>
```

Comment:
adjust the block <a> with the content in the next line within a block with correct indentation